packages/react-native-renderer/src/__tests__/ReactNativeAttributePayload-test.internal.js

```diff
@@ -204,6 +204,19 @@ describe('ReactNativeAttributePayload', () => {
     ).toEqual({foo: null}); // this should ideally be null. heuristic.
   });
 
+  it('handles attributes defined multiple times', () => {
+    const validAttributes = {foo: true, style: {foo: true}};
+    expect(diff({}, {foo: 4, style: {foo: 2}}, validAttributes)).toEqual({
+      foo: 2,
+    });
```

Comment:
Wow really?

Reply:
Yuup.. which means code like https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/Components/View/View.js#L103 is unnecessary

For most native props on View etc, we allow them as valid attributes as a direct prop and as style..

Reply:
Oh wait is this only for `style` properties, I remember that now. 

But what happens if you have custom native component that defines a `backgroundColor` prop?